tutorials/2_Gazebo_and_ROS.ipynb

```diff
@@ -0,0 +1,1330 @@
+    "\n",
+    "\n",
+    "\n",
+    "  <!-- caster wheel radius and height -->\n",
+    "  <!-- caster wheel mass -->\n",
+    "  <property name=\"caster_f_height\" value=\"0.04\" /> \n",
+    "  <property name=\"caster_f_radius\" value=\"0.025\" /> \n",
+    "  <property name=\"caster_f_mass\" value=\"0.5\" /> <!-- in kg-->\n",
+    "\n",
+    "\n",
+    "\n",
+    "  <!-- caster wheel radius and height -->\n",
+    "  <!-- caster wheel mass -->\n",
+    "  <property name=\"caster_b_height\" value=\"0.04\" /> \n",
+    "  <property name=\"caster_b_radius\" value=\"0.025\" /> \n",
+    "  <property name=\"caster_b_mass\" value=\"0.5\" /> <!-- in kg-->\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p>Every file must contain the robot tag and everything we put in them should be in this tag.</p>\n",
+    "\n",
+    "<p>Now we want to add a circular base for our robot. Insert this within the robot tag of diff_wheeled_robot_description.xacro:</p>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "<!-- BASE-LINK -->\n",
+    "  <!--Actual body/chassis of the robot-->\n",
+    "  <link name=\"base_link\">\n",
+    "    <inertial>\n",
+    "      <mass value=\"${base_mass}\" />\n",
+    "      <origin xyz=\"0 0 0\" />\n",
+    "      <!--The 3x3 rotational inertia matrix. -->\n",
+    "        <cylinder_inertia  m=\"${base_mass}\" r=\"${base_radius}\" h=\"${base_height}\" />\n",
+    "\n",
+    "    </inertial>    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<ul>\n",
+    "    \n",
+    "<li>inertial tag (optional): The inertial properties of the link.</li>\n",
+    "<li>origin tag (optional: defaults to identity if not specified): This is the pose of the inertial reference frame, relative to the link reference frame. The origin of the inertial reference frame needs to be at the center of gravity. The axes of the inertial reference frame do not need to be aligned with the principal axes of the inertia.</li>\n",
+    "\n",
+    "<li>xyz tag (optional: defaults to zero vector): Represents the x,y,z offset.</li>\n",
+    "\n",
+    "<li>rpy (optional: defaults to identity if not specified): Represents the fixed axis roll, pitch and yaw angles in radians.</li>\n",
+    "<li>mass tag: The mass of the link is represented by the value attribute of this element.</li>\n",
+    "<li>inertia tag: The 3x3 rotational inertia matrix, represented in the inertia frame. Because the rotational inertia matrix is symmetric, only 6 above-diagonal elements of this matrix are specified here, using the attributes ixx, ixy, ixz, iyy, iyz, izz,</li>\n",
+    "</ul>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "    <visual>\n",
```

Comment:
This model is getting really complex really quick. Build a simpler model during the tutorial, and use the full robot model for the autonomous operation example. But don't explain the full robot model.